src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$cell-color: #eee;
+$gap: 1px;
+$count-of-days: 31;
```

Comment:
do you need it
u only use it once

Reply:
@VolodymyrKirichenko about condition, in the first version of my code it wasn't, but when i compiled my file to .css, it was a mistake, because near the 'px' was a number zero. And i don't understand how i can avoid it... Can u help me pls with this?

Reply:
> @VolodymyrKirichenko about condition, in the first version of my code it wasn't, but when i compiled my file to .css, it was a mistake, because near the 'px' was a number zero. And i don't understand how i can avoid it... Can u help me pls with this?

@PikalovAndrey `($dayBlockSize + $dayBlockGap) * $dayValue`;
if the day is 0 - the margin will also be 0)